src/rules/deprecationRule.ts

```diff
@@ -57,11 +57,11 @@ class Walker extends Lint.ProgramAwareRuleWalker {
     }
 
     for (const d of decSym.getDeclarations()) {
+      let commentNode: ts.Node = d;
```

Comment:
Yes, the comment included the string "@deprecated", so the declaration of `commentNode` was considered deprecated.